ci-operator/config/windup/windup-ui-tests/windup-windup-ui-tests-main__mtr-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,57 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: dockerfiles/interop/Dockerfile
+  to: mtr-runner
+releases:
+  initial:
```

Comment:
This initial block does not appear to be needed. It appears to just be needed when we are concerned with testing upgrades.

If you search `RELEASE_IMAGE_INITIAL` in the [ci-operator-reference](https://steps.ci.openshift.org/ci-operator-reference) you'll see:
``
an 'initial' name will be exposed as $RELEASE_IMAGE_INITIAL.

If upgrade is true, RELEASE_IMAGE_INITIAL will be used as the initial payload and the installer image from that will be upgraded.
``

From a job with both being included we use latest: `Imported release 4.13.0-0.nightly-2023-02-17-090603 created at 2023-02-17 09:07:40 +0000 UTC with 183 images to tag release:latest `

So initial is never used. I believe that we can remove this section. It passes `make update` and I'm running a test of just using latest now.

Reply:
Oh, that's good to know. I was curious why we included that. Just hadn't tried removing it yet. I have made the changes locally, let me know if you test works. If so, I'll push the changes.